documentation/specs/Stepper.md

```diff
@@ -0,0 +1,104 @@
+# `Stepper` Component Specification
+
+## Overview
+
+A `Stepper` component is used to indicate progress as the user goes through a multi-step process.
+
+### Use Cases
+
+- Guide a user through a large form where parts of the form can be separated into logical steps.
+- Guide a user through a sequence of steps where the next step depends on data from the previous step.
+
+### Features
+
+- The `orientation` property can be used to display a vertical or horizontal stepper.
+- The `color` property can be used to display a stepper with primary or inverse colors; inverse is suitable on darker backgrounds.
+- The step sequence is non-linear, which means users can navigate to previously completed steps.
+- On small screens, the step label is not visible.
+- A step can either be active, accessible, completed, or disabled.
+
+---
+
+## Design
+
+In contrast to most Easy UI components, the `Stepper` component does not have an immediate `react-aria` primitive to build on top of.
+
+The `Stepper` component has two considerations: The outer container and individual steps. The outer container will handle orientation and spacing concerns with each individual step with respect to another. Individual steps will be represented as `Stepper.Step` and each `Stepper.Step` renders a `Stepper.StepButton` and behind the scenes a `Stepper.StepSeparator`. The `Stepper.StepButton` will make use of the `UnstyledButton` component and the clickable area will not only include the label, but a left aligned icon as well. Representing each step as a button is necessary to support a non-linear step sequence where previously completed steps can be clicked and returned to. The `Stepper.StepSeparator` will not be exposed to the consumer, and will be omitted for the last step in the collection.
+
+### API
+
+```ts
+type StepperColor = "primary" | "inverse";
+type StepperOrientation = "horizontal" | "vertical";
+
+type Stepper = {
+  /**
+   * Color scheme that applies to stepper in various states.
+   */
+  color?: StepperColor;
+  /**
+   * Orientation of stepper, vertical replaces arrow icon with line.
+   */
+  orientation?: StepperOrientation;
```

Comment:
i assume defaults for these are `primary` and `horizontal`?

Reply:
ah yes, updated with defaults